src/js/components/hooks/useSetTimeout.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+import {useEffect, useRef} from "react"
+
+export default function useSetTimeout() {
```

Comment:
This is for when I set a timeout in a click handler. If the component is unmounted, the timeout will still fire and might ask the component to "re-render". But since the component is unmounted, react throws this error:

```
Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method
```